app/controllers/planning_applications_controller.rb

```diff
@@ -135,35 +143,65 @@ def recommend
     @planning_application.assign_attributes(params.require(:planning_application).permit(:decision, :public_comment))
     @recommendation.assign_attributes(params.require(:recommendation).permit(:assessor_comment).merge(assessor: current_user))
     if @planning_application.save && @recommendation.save
-      @recommendation.update(submitted: true)
+      @planning_application.assess!
       redirect_to @planning_application
     else
       render :recommendation_form
     end
   end
 
-  def submit_recommendation; end
+  def submit_recommendation
+    respond_to do |format|
+      if @planning_application.can_submit_recommendation?
+        format.html { render :submit_recommendation }
+      else
+        format.html { render plain: "Not Found", status: :not_found }
+      end
+    end
```

Comment:
is there any particular reason of using `respond_to` here since you're returning always html?

Reply:
https://blog.arkency.com/2016/07/respond-to-format-is-useful-even-without-multiple-formats/

Reply:
I'm just quite used to this style of rendering layouts too.